control_software/scripts/hera_upload_librarian.py

```diff
@@ -42,7 +42,13 @@ def process_next(f):
     instances = client.search_instances(query)  # takes a long time
     if len(instances['results']) == 0:
         # this file is not in the librarian, so add it
-        full_path = os.path.join(DATA_DIR, f)
+        try:
+            # this assumes the entry is in a format like '2460425/zen.2460425.51498.sum.uvh5'
+            int_jd = int(f.split('/')[0])
+        except:
```

Comment:
I'm not usually a fan of bare `except` statements. I also think it's okay to break backwards compatibility, because we (probably) won't be going back to how we had it before.

Reply:
`/mnt/sn1` is already hard-coded, so maybe we make it `/mnt/sn1/dataX` and we replace the X by 1 or 2 algorithmically...